.gitignore

```diff
@@ -26,5 +26,6 @@
 /.sass-cache
 # Bazelisk version file
 .bazelversion
+.bazeliskrc
```

Comment:
Please remove (I'm not sure what our stance is on adding additional entries to .gitignore, but if you really want to add it, this is better discussed in a separate PR.)